cmd/download.go

```diff
@@ -132,7 +132,7 @@ func runDownload(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 		}
 	}
 
-	solution := workspace.Solution{
+	solution := workspace.ExerciseMetadata{
```

Comment:
Would it be too much of a change, confusion, if we renamed this to metadata? If we can't change the name with out changing a ton of code right now what do you think about adding a `TODO`?

Reply:
I went back and forth on `Metadata` vs `ExerciseMetadata` and was worried that `Metadata` was too unspecific. I like the shorter name better, though, so I'll go with that.

Reply:
Sorry for not being explicit when I mentioned renaming this. I meant just changing `solution` to `metadata`. I'm on the fence about the name. `ExerciseMetadata` is clearer, but longer.